hutch_python/load_conf.py

```diff
@@ -128,17 +130,20 @@ def load_conf(conf, hutch_dir=None):
 
     Parameters
     ----------
-    conf: ``dict``
+    conf : ``dict``
```

Comment:
May as well fix up the types here while you're touching the line:
```suggestion
    conf : dict
```

Reply:
Forgot about this one, it looked correct when I read through it